dataikuapi/dssclient.py

```diff
@@ -629,6 +629,59 @@ def create_global_api_key(self, label=None, description=None, admin=False):
         key = resp.get('key', '')
         return DSSGlobalApiKey(self, key)
 
+    def delete_global_api_key(self, key):
+        """
+        Delete a specific global API Key
+
+        Note: this call requires an API key with admin rights
+        """
+        resp = self._perform_http(
+            "DELETE", "/admin/globalAPIKeys/" + key)
+        if resp.status_code != 200:
+            raise Exception('API key creation failed : %s' % resp.json())
```

Comment:
`dataikuapi.dss.admin.DSSGlobalApiKey` already has a `delete` method. For consistency, this method should be removed.